GCTC-NTGC/gc-digital-talent
# Admin - Update UserPage PageHeader icon

Resolves #3308.

### Note
[Quick fix](https://github.com/GCTC-NTGC/gc-digital-talent/issues/3308#issuecomment-1180525519), use same icon as corresponding AdminSideMenu icon.

### Steps to test

1. Change directory to `frontend/admin`
2. In the aforementioned directoy, run `npm run dev`
3. Log in to admin application as admin
4. Navigate to **/en/admin/users** page in browser
5. Verify both the side menu item and the page header for Users are the same
